install/install.go

```diff
@@ -559,7 +563,28 @@ func (k *K8sInstaller) restartUnmanagedPods(ctx context.Context) error {
 
 }
 
+func (k *K8sInstaller) printVersions() error {
+	versions, err := helm.ListVersions()
+	if err != nil {
+		return err
+	}
+	// Iterate backwards to print the newest version first.
+	for i := len(versions) - 1; i >= 0; i-- {
+		if versions[i] == defaults.Version {
+			fmt.Println(versions[i], "(default)")
+		} else {
+			fmt.Println(versions[i])
+		}
+	}
+	return err
+}
+
 func (k *K8sInstaller) Install(ctx context.Context) error {
+	// If --list-versions flag is specified, print available versions and return.
+	if k.params.ListVersions {
+		return k.printVersions()
```

Comment:
silly nit, but the flag uses "list" but the function is called "print". Maybe be consistent? silly nit though.

Reply:
it's good to be consistent 👍